### PR TITLE
docs: :memo: Document workaround for server timeout issue in README

### PR DESCRIPTION
**Pull request by isalmanhaider**
Add a section to the README detailing the server timeout issue experienced by some users and the recommended command (`composer drupal:run-server`) to restart the server and mitigate the problem. This update provides clarity and support for developers encountering this issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,25 @@ composer create-project drupal/starshot-project --repository='{"type":"vcs","url
...
 
+## Known Issues and Workarounds
+
+While Starshot aims to provide a seamless setup and usage experience, there are occasional hiccups that might occur during development. Below is a common issue and its recommended workaround.
+
+### Server Timeout Issue
+Some users have reported experiencing a timeout issue after logging into Drupal Starshot, particularly when the system remains idle for some time. This might lead to the server stopping unexpectedly.
+
+**Symptom:**
+The process may exceed the timeout limit, leading to an error similar to:
```
Comment (phenaproxima) [inline README.md:26]:
```suggestion
You may see an error like this:
```

I'd also remove the "Symptom" line. I'm not sure it really adds anything.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,25 @@ composer create-project drupal/starshot-project --repository='{"type":"vcs","url
...
+```
+The process "test -n "$CI" || composer drupal:run-server" exceeded the timeout of 300 seconds.
+```
+
+**Workaround:**
```
Comment (phenaproxima) [inline README.md:31]: We can remove this line. The next sentence makes clear that it's a workaround.

File: README.md [hunk 3]
```diff
@@ -15,6 +15,25 @@ composer create-project drupal/starshot-project --repository='{"type":"vcs","url
...
+If you encounter this timeout issue, you can restart the server using the following command:
```
Comment (phenaproxima) [inline README.md:32]:
```suggestion
If you encounter this, you can restart the server using the following command:
```
A little less verbose.

**Review by phenaproxima — CHANGES_REQUESTED** [review]
This looks great. I have a few requests, but otherwise I'd be happy to merge this once those are addressed.

File: README.md [hunk 4]
```diff
@@ -15,6 +15,25 @@ composer create-project drupal/starshot-project --repository='{"type":"vcs","url
 ```
 This one command will install Starshot and open it in a web browser for you to play with. You'll get all the modules and themes listed below, pre-configured.
 
+## Known Issues and Workarounds
+
```
[the remark this answers is not in the document]
Comment (isalmanhaider) [reply README.md]: Done

[the remark this answers is not in the document]
Comment (isalmanhaider) [reply README.md]: Done

**Comment by isalmanhaider** [comment]
@phenaproxima 

Thanks for the feedback! I've made the requested changes. 
Please let me know if there's anything else needed for the merge.

**Comment by phenaproxima** [comment]
Only one thing - can you remove the line that says "Symptom"?

Otherwise this is good to go.

**Comment by isalmanhaider** [comment]
@phenaproxima Removed the 'Symptom' heading/line.

**Review by phenaproxima — APPROVED** [review]

**Pull request merged**